packages/playground/src/app.jsx

```diff
@@ -6,6 +6,7 @@ import Form, { withTheme } from "@rjsf/core";
 import { shouldRender } from "@rjsf/utils";
 import DemoFrame from "./DemoFrame";
 import localValidator from "@rjsf/validator-ajv6";
```

Comment:
Reorder this so all external imports are first
```suggestion
import localValidator from "@rjsf/validator-ajv6";

import DemoFrame from "./DemoFrame";
```